alerta-covid/app/controllers/api/v1/symptoms_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::SymptomsController < Api::V1::ApiController
+    skip_before_action :verify_authenticity_token
+
+    def index
+        symptom = Symptom.all
+        render json: symptom, status: :ok
+    end
+
+end
```

Comment:
Check your indentation here
Would be nice if you use a linter like rubocop or standard-rb